src/components/content-switcher/_content-switcher.scss

```diff
@@ -27,6 +27,15 @@
     border: 1px solid $brand-01;
     color: $brand-01;
 
+    > svg {
+      margin-right: 0.5rem;
```

Comment:
We have brand new spacing variables that have been added recently. You can see them here, https://github.com/carbon-design-system/carbon-components/blob/master/src/globals/scss/_spacing.scss  So it would be great to change this to `margin-right: $spacing-xs;'

Reply:
thanks @alisonjoseph for the feedback, resubmitting with both these changes.